recipe/run_test.sh

```diff
@@ -18,6 +18,12 @@ $FC -O3 -fopenmp -ffast-math -o maths maths.f90 -v
 ./maths
 rm -f maths
 
+$FC -c print_hello.f90 -o print_hello.o
+${CXX} -c main.cpp -o main.o
+${CXX} main.o print_hello.o -o main ${LDFLAGS} -lgfortran
+${FC} main.o print_hello.o -o main ${LDFLAGS} -lc++
```

Comment:
We should run test programs right?

Reply:
Just trying to trigger a link failure here

Reply:
Fair. It might be good to do anyways.